components/application/widgets/NoTask.js

```diff
@@ -16,14 +16,21 @@ const NoTask = (props) => {
       break;
     case "inbox":
       content = " All your team's tasks are organized in the right place.";
-      image = "/images/notask_inbox.jpg";
+      image = "/images/notask-inbox.jpg";
```

Comment:
I am curious about the intention behind the changes made in this section. If there is a rationale, I believe we should standardize the naming convention for all files in the 'image' directory. Currently, there is inconsistency with some files using hyphens, others using underscores, and some following camelCase. It appears quite disorganized. Perhaps we can create an issue to address and unify this aspect.

Reply:
I noticed that the file name you used before contained multiple situations, so I decided to use a horizontal line in the middle. However, if you feel disorganized, you can unify the meaning.